SpreadCheetah/Validations/ValidationType.cs

```diff
@@ -6,5 +6,7 @@ internal enum ValidationType
     Integer,
     Decimal,
     TextLength,
-    List
+    List,
+    DateOnly,
+    DateTime
```

Comment:
You need to also update `DataValidationXml` to handle the enum value.

Reply:
Could you be more specific as to where and how I should update DataValidationXml?

Reply:
Would it be the `TryWriteType `method that needs to be updated?

Reply:
Yes 🙂

Reply:
Do you have any documentation that explains the different data types (and respective strings) supported by the method? I have tried the following options:

1. `ValidationType.DateTime => "type=\"datetime\" "u8.TryCopyTo(bytes, ref bytesWritten),`
2. `ValidationType.DateTime => "type=\"date\" "u8.TryCopyTo(bytes, ref bytesWritten),`


So far, it seems that the latter is the correct choice. Nevertheless, I get uneasy do to its name which implies solely in a DateOnly object; therefore, losing the time property of the Datetime element.

Reply:
The documentation is basically the Open XML specification, but it can be a bit hard to read. So I suggest you instead:
1. Create the validation in Excel
2. Save it to an `.xlsx` file
3. Rename the `.xlsx` file to `.zip`
4. Unzip that file
5. Look at the content of the `xl/worksheets/sheet1.xml` file

Reply:
Dear Sveinungf, thank you for the feedback. As it seems, after unzip, within the docProps/core.xml file, I managed to retrieve the following type: dcterms:W3CDTF. I am not sure whether or not this is the correct type. If we look at the ./xl/worksheets/sheet1.xml file, we can find the "date" type. Therefore, so far, I would keep the "date" type, instead of the "dcterms:W3CDTF".

Reply:
After some trials, the `ValidationType.DateTime => "type=\"date\" "u8.TryCopyTo(bytes, ref bytesWritten),` option was the right choice. 

All tests are passing.